doc/api/module.md

```diff
@@ -1161,6 +1161,7 @@ The final value of `format` must be one of the following:
 
 | `format`     | Description                    | Acceptable types for `source` returned by `load`                           |
 | ------------ | ------------------------------ | -------------------------------------------------------------------------- |
+| `'addon'`    | Load a Node.js addon           | { `null` }                                                                 |
```

Comment:
```suggestion
| `'addon'`    | Load a Node.js addon           | {null}                                                                 |
```

Reply:
The line below quoted `null` with backticks, is there any rule about this?

https://github.com/nodejs/node/blob/5a868b5b30e551d375c73d65013883e6078bba84/doc/api/module.md?plain=1#L1165